packages/components/src/base-field/test/index.tsx

```diff
@@ -8,9 +8,13 @@ import { render, screen } from '@testing-library/react';
  */
 import { useBaseField } from '../index';
 import { View } from '../../view';
+import type { WordPressComponentProps } from '../../ui/context';
+import type { BaseFieldProps } from '../types';
 
-const TestField = ( props ) => {
-	return <View { ...useBaseField( props ) } />;
+const TestField = (
+	props: Omit< WordPressComponentProps< BaseFieldProps, 'div' >, 'children' >
```

Comment:
Nit: I don't think it's necessary to use `WordPressComponentProps` in this test, `Omit< BaseFieldProps, 'children' >` should suffice

Reply:
Done in https://github.com/WordPress/gutenberg/pull/45712/commits/dc1ffdc917712a3c405e5fc6d5b5ad15c330dd89